packages/arb-token-bridge-ui/src/components/App/App.tsx

```diff
@@ -60,66 +54,15 @@ const rainbowkitTheme = merge(darkTheme(), {
     accentColor: 'var(--blue-link)'
   },
   fonts: {
-    body: "'Space Grotesk', sans-serif"
+    body: 'Roboto, sans-serif'
   }
 } as Theme)
 
-const AppContent = (): JSX.Element => {
-  const [{ sourceChain }] = useNetworks()
+const ArbTokenBridgeStoreSyncWrapper = (): JSX.Element | null => {
+  const actions = useActions()
   const {
-    app: { connectionState }
+    app: { selectedToken }
   } = useAppState()
-
-  useSyncQueryParamsToTestnetMode()
-
-  const headerOverridesProps: HeaderOverridesProps = useMemo(() => {
-    if (isNetwork(sourceChain.id).isTestnet) {
-      return {
-        imageSrc: 'images/HeaderArbitrumLogoTestnet.webp',
-        className: 'lg:bg-ocl-blue'
-      }
-    }
-
-    return {
-      imageSrc: 'images/HeaderArbitrumLogoMainnet.svg',
-      className: 'lg:bg-black'
-    }
-  }, [sourceChain.id])
-
-  if (connectionState === ConnectionState.NETWORK_ERROR) {
```

Comment:
we never set the `connectionState` to `ConnectionState.NETWORK_ERROR` in the code base so this case will never be hit